hooks/pre/10-reset_feature.rb

```diff
@@ -19,8 +19,8 @@ def foreman_installed?
 end
 
 def stop_services
-  Kafo::KafoConfigure.logger.info 'Ensuring services httpd and foreman-tasks are stopped.'
-  Kafo::Helpers.execute('service httpd stop && service foreman-tasks stop')
+  Kafo::KafoConfigure.logger.info 'Stopping services'
+  Kafo::Helpers.execute('katello-service stop --exclude postgresql'
```

Comment:
@ekohl not really because we do it all here:

https://github.com/Katello/katello-installer/blob/master/hooks/pre/10-reset_feature.rb#L46-L55